src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1667,8 +1692,10 @@ namespace ReportSizingManager {
 							DesVolFlow = UnitarySysEqSizing( CurSysNum ).HeatingAirVolFlow;
 						} else {
 							if ( CurDuctType == Main ) {
-								if ( FinalSysSizing( CurSysNum ).SysAirMinFlowRat > 0.0 ) {
+								if ( FinalSysSizing( CurSysNum ).SysAirMinFlowRat > 0.0 && !DataDesicRegCoil ) {
 									DesVolFlow = FinalSysSizing( CurSysNum ).SysAirMinFlowRat*FinalSysSizing( CurSysNum ).DesMainVolFlow;
+								} else if ( DataDesicRegCoil ) {
+									DesVolFlow = FinalSysSizing( CurSysNum ).DesMainVolFlow;
```

Comment:
OK. I moved the DataDesicRegCoil flag check as the first.  Without this flag, the flow rate could be sizing for the minimum flow.  That is not what we want because the FinalSysSizing( CurSysNum ).SysAirMinFlowRat can be > 0.0.